### PR TITLE
Restore tracked tag counts in sidebar

### PR DESCRIPTION
**Pull request by jwbay**
Leaves the search bar dropdown out in the cold for now

### CODE REVIEW

File: Extensions/classic_tags.js [hunk 1]
```diff
@@ -9,6 +9,7 @@ XKit.extensions.classic_tags = new Object({
 
 	running: false,
 	slow: true,
+	apiKey: XKit.api_key, //*"fuiKNFp9vQFvjLNvx4sUwti4Yb5yGutBN4Xh10LXZhhRKjWlV4",**//
```
Comment (Wolvan) [inline Extensions/classic_tags.js]: Please remove the comment

File: Extensions/classic_tags.js [hunk 2]
```diff
@@ -82,6 +83,97 @@ XKit.extensions.classic_tags = new Object({
 		$(this).attr("href", newHref);
 	},
 
+	get_post_timestamp: function(blog_name, post_id) {
+		var self = this;
+		var api_url = "https://api.tumblr.com/v2/blog/" + blog_name + "/posts" + "?api_key=" + self.apiKey + "&id=" + post_id;
+		var promise = $.Deferred();
+
+		function fail() {
+			XKit.console.add('Unable to fetch timestamp for post ' + post_id);
```
Comment (finagle29) [inline Extensions/classic_tags.js]: Please don't use `XKit.console.add`, perhaps `XKit.notifications.add(message, type, sticky, callback)` instead?

Comment (nightpool) [inline Extensions/classic_tags.js]: Either expose the error to the user (bad in this case I think) or just use
console.log.

Comment (jwbay) [inline Extensions/classic_tags.js]: Opted for console.log, users aren't going to be able to do anything about these errors

File: Extensions/classic_tags.js (hunk 1 above)
Comment (jwbay) [inline Extensions/classic_tags.js]: Done! I saw it in so many other places I actually thought it was supposed to be there.

**Comment by 00Davo** [comment]
This is working really nicely! :+1: Let's get this merged, unless anyone's noticed something drastically wrong with it.

Pretty sure All of Tumblr are waiting for this fix, so. Yeah. :wink:

**Comment by Wolvan** [comment]
@homu r=00Davo

**Pull request merged**